.github/labeller.yml

```diff
@@ -0,0 +1,52 @@
+labels:
```

Comment:
IMO, if this is going to be ingested and converted in Nuke, we should just define them in Nuke, instead of having to parse and interpret from YAML. If you defined them in Nuke directly, you get extra IDE support for the regexes, and don't limit yourself to what can be defined in YAML. I don't think it gains a lot.

Reply:
Good point. I didit as a reflex to split conf and code. But I don't mind either way.